frontend/constants/serviceTemplates.ts

```diff
@@ -358,8 +358,8 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
       STAKING_ACTIVITY_CHECKER_CONTRACT_ADDRESS: {
         name: 'Staking activity checker contract address',
         description: '',
-        value: '',
-        provision_type: EnvProvisionType.COMPUTED,
+        value: '0x07bc3C23DbebEfBF866Ca7dD9fAA3b7356116164',
+        provision_type: EnvProvisionType.FIXED,
```

Comment:
is this just temporary? a quick solution?

Reply:
I found it hardcoded in the modius-quickstart, so kept it fixed here also. Should it be computed? If yes, then how?

Reply:
merging right now, to unblock Tanya. But lmk

Reply:
The staking contract should define it I think.